test/functional/cmdLineTests/xlogTests/xlog.xml

```diff
@@ -0,0 +1,251 @@
+		<output regex="no" type="required">verbosegc</output>
+		<output regex="no" type="failure">JVMJ9VM085</output>
+	</test>
+
+	<test id="-Xlog:gc,any=off">
+		<command>$EXE$ $ENABLEXLOG$ -Xlog:gc,any=off -version</command>
+		<output regex="no" type="success">openjdk version</output>
+		<output regex="no" type="required">JVMJ9VM007W</output>
+		<output regex="no" type="required">verbosegc</output>
+		<output regex="no" type="failure">JVMJ9VM085</output>
+	</test>
+
+	<test id="-Xlog:gc">
+		<command>$EXE$ $ENABLEXLOG$ -Xlog:gc -version</command>
+		<output regex="no" type="success">openjdk version</output>
+		<output regex="no" type="required">verbosegc</output>
+		<output regex="no" type="failure">JVMJ9VM085</output>
+		<output regex="no" type="failure">JVMJ9VM007</output>
+	</test>
+
+	<test id="-Xlog:gc:">
+		<command>$EXE$ $ENABLEXLOG$ -Xlog:gc: -version</command>
+		<output regex="no" type="success">openjdk version</output>
+		<output regex="no" type="required">verbosegc</output>
+		<output regex="no" type="failure">JVMJ9VM085</output>
+		<output regex="no" type="failure">JVMJ9VM007</output>
+	</test>
+
+	<test id="-Xlog:gc:stderr">
+		<command>$EXE$ $ENABLEXLOG$ -Xlog:gc:stderr -version</command>
+		<output regex="no" type="success">openjdk version</output>
+		<output regex="no" type="required">verbosegc</output>
+		<output regex="no" type="failure">JVMJ9VM085</output>
+		<output regex="no" type="failure">JVMJ9VM007</output>
+	</test>
+
+	<test id="-Xlog:gc:stderr">
+		<command>$EXE$ $ENABLEXLOG$ -Xlog:gc:stderr: -version</command>
+		<output regex="no" type="success">openjdk version</output>
+		<output regex="no" type="required">verbosegc</output>
+		<output regex="no" type="failure">JVMJ9VM085</output>
+		<output regex="no" type="failure">JVMJ9VM007</output>
+	</test>
+
+	<test id="-Xlog:gc:stderr:any">
+		<command>$EXE$ $ENABLEXLOG$ -Xlog:gc:stderr:any -version</command>
+		<output regex="no" type="success">openjdk version</output>
+		<output regex="no" type="required">verbosegc</output>
+		<output regex="no" type="required">JVMJ9VM007W</output>
+		<output regex="no" type="failure">JVMJ9VM085</output>
+	</test>
+
+	<test id="-Xlog:gc:stdout">
+		<command>$EXE$ $ENABLEXLOG$ -Xlog:gc:stdout -version</command>
+		<output regex="no" type="success">openjdk version</output>
+		<output regex="no" type="required">verbosegc</output>
+		<output regex="no" type="required">JVMJ9VM007W</output>
+		<output regex="no" type="failure">JVMJ9VM085</output>
+	</test>
+
+	<test id="-Xlog:gc:gclog">
+		<command>$EXE$ $ENABLEXLOG$ -Xlog:gc:gclog -version</command>
+		<output regex="no" type="success">openjdk version</output>
+		<output regex="no" type="failure">verbosegc</output>
+		<output regex="no" type="failure">JVMJ9VM085</output>
+		<output regex="no" type="failure">JVMJ9VM007</output>
+	</test>
+
+	<test id="-Xlog:gc:gclog check file">
+		<command>cat gclog</command>
+		<output regex="no" type="success">verbosegc</output>
+	</test>
```

Comment:
Are these tests guaranteed to be run sequentially (not in parallel)?

Reply:
Yes they run sequentially and test are written to take advantage of that, otherwise many of the suites would have problems.